### DISCUSSION TITLE
Generate structured synthetic data based on json schema using open source llm

### DISCUSSION THREAD

**Discussion by rajasingh012** [microsoft/semantic-kernel#9334] [General]
Using c#, I am working to generate structured synthetic data based on Json schema using open source llm, How to start, any links or project as reference.

**Comment by sophialagerkranspandey** [comment]
@esttenorio could you take a look at this issue and share any relevant samples? Thanks!

**Comment by esttenorio** [comment]
As part of the new [SK Processes Framework](https://learn.microsoft.com/en-us/semantic-kernel/frameworks/process/process-framework), in the near future there will be samples that cover up ways on how to fill up a structured data using JSON schemas both in python and .NET.

For now, there's SK Process sample that is a simple example that provides a starting point on a way to fill up structured data:
https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/GettingStartedWithProcesses/Step02/Steps/CompleteNewCustomerFormStep.cs

It's not the same since it does not make use of a JSON schema but it does make use of a structured object to fill up and only move on when it is completed.

**Comment by rajasingh012** [comment]
Thanks @rvinothrajendran for the discussion and valuable inputs.
